loom/common/core/performance.cpp

```diff
@@ -530,16 +547,21 @@ static void LoomProfilerEntryDumpRecurse(LoomProfilerEntry *data, char *buffer,
     else
     {
         // dump out this one:
-        lmLogError(gProfilerLogGroup, "%7.3f %7.3f %8d %s%s",
+        lmLogError(gProfilerLogGroup, "%8.3f %8.3f %8.3f %8.3f %8.3f %8d %s%s",
                    data->mRoot == NULL ? 100.0 : 100.0 * data->mTotalTime / totalTime,
                    data->mRoot == NULL ? 100.0 : 100.0 * (data->mTotalTime - data->mSubTime) / totalTime,
+                   data->mTotalTime / (1000.0 * 1000.0 * (data->mInvokeCount > 0 ? data->mInvokeCount : 1)),
+                   data->mMaxTime / (1000.0 * 1000.0),
```

Comment:
Hmm, it shouldn't (Ctrl+F for "Turbo Boost"): https://msdn.microsoft.com/en-us/library/windows/desktop/dn553408%28v=vs.85%29.aspx
I didn't see unreasonable times when I was testing it.